src/community/geogig/src/test/java/org/geogig/geoserver/config/RepositoryManagerTest.java

```diff
@@ -156,7 +156,7 @@ public void testCreateRepoUnsupportedURIScheme() {
         } catch (IllegalArgumentException e) {
             // expected
             assertEquals(
-                    "No repository initializer found capable of handling this kind of URI: unknown://repo1",
+                    "No repository initializer found capable of handling this kind of URI: unknown",
```

Comment:
This looks like it represents a change in behaviour... Do you know why the error message changed?